docs/eventing/broker-trigger.md

```diff
@@ -55,39 +55,39 @@ spec:
 
 ## Usage
 
-### ClusterChannelProvisioner
+### Channel
 
-`Broker`s use their `spec.channelTemplate` to create their internal `Channel`s,
+`Broker`s use their `spec.channelTemplateSpec` to create their internal `Channel`s,
 which dictate the durability guarantees of events sent to that `Broker`. If
-`spec.channelTemplate` is not specified, then the
-[default provisioner](https://www.knative.dev/docs/eventing/channels/default-channels/)
+`spec.channelTemplateSpec` is not specified, then the
+[default channel](./channels/default-channels.md)
 for their namespace is used.
 
 #### Setup
 
-Have a `ClusterChannelProvisioner` installed and set as the
-[default provisioner](https://www.knative.dev/docs/eventing/channels/default-channels/)
+Have a `Channel` CRD installed and set as the
+[default channel](./channels/default-channels.md)
```

Comment:
```suggestion
default channel
```

Since this is linked directly above, we don't need this second one (to avoid making it look like it goes somewhere else)

Reply:
Done, thanks!